packages/@sanity/base/src/components/validation/listItem.tsx

```diff
@@ -0,0 +1,75 @@
+import {Marker, Path} from '@sanity/types'
+import React, {useCallback, useMemo} from 'react'
+import {WarningOutlineIcon, ErrorOutlineIcon} from '@sanity/icons'
+import {Box, Text, MenuItem, Stack, Flex, Theme, CardTone} from '@sanity/ui'
+import styled from 'styled-components'
+
+interface ValidationListItemProps {
+  marker: Marker
+  onClick?: (path?: Path) => void
+  path: string
+  // showLink?: boolean
```

Comment:
What is `showLink` and why is it "commented out"? Are we planning on adding it later? Maybe we could either remove it or add a comment with an explanation?

Reply:
Not sure! It was how it was in the original component so I didn't want to just remove it

Reply:
If it's not used anywhere, I'd say remove it :)